docs/DeveloperGuide.md

```diff
@@ -405,6 +408,9 @@ Use case ends.
 * 2a. No matching clients found.
     * 2a1. Clientell informs the financial consultant that no matches were found.
     * Use case ends.
+* 2b. The financial consultant uses the command in the transaction list view.
+    * 2b1. Clientell shows an error message.
+    * Use case ends.
```

Comment:
I feel like the sequence is a little strange, wouldn't the financial consultant already be in the transaction view before entering the command? Maybe this is separate use case or can have a precondition for this use case that user is in client view.
![image](https://github.com/user-attachments/assets/ff467946-64af-45f8-8e2e-6f115972e407)

Reply:
Not sure if preconditions and guarantees are necessary for each use case?

Reply:
but this is the way that we check if the command is valid?